runtime/bindings/python/src/compatibility/ngraph/opset8/ops.py

```diff
@@ -417,3 +417,108 @@ def gather_nd(
+                                    Range of values: False - max_size is ignored
+                                                     True  - max_size is used
+                                    Default value: True
+                                    Required: no
+    * fixed_ratio                   This is an aspect ratio of a box.
+                                    Range of values: a list of positive floating-point numbers
+                                    Default value: None
+                                    Required: no
+    * fixed_size                    This is an initial box size (in pixels).
+                                    Range of values: a list of positive floating-point numbers
+                                    Default value: None
+                                    Required: no
+    * density                       This is the square root of the number of boxes of each type.
+                                    Range of values: a list of positive floating-point numbers
+                                    Default value: None
+                                    Required: no
+    * min_max_aspect_ratios_order   The flag that denotes the order of output prior box.
+                                    Range of values: False - the output prior box is in order of
+                                                             [min, aspect_ratios, max]
+                                                     True  - the output prior box is in order of
+                                                             [min, max, aspect_ratios]
+                                    Default value: True
+                                    Required: no
+    Example of attribute dictionary:
+    @code{.py}
+        # just required ones
+        attrs = {
+            'offset': 85,
+        }
+        attrs = {
+            'offset': 85,
+            'flip': True,
+            'clip': True,
+            'fixed_size': [32, 64, 128]
+        }
+    @endcode
+    Optional attributes which are absent from dictionary will be set with corresponding default.
+    """
+    requirements = [
+        ("offset", True, np.floating, is_non_negative_value),
+        ("min_size", False, np.floating, is_positive_value),
+        ("max_size", False, np.floating, is_positive_value),
+        ("aspect_ratio", False, np.floating, is_positive_value),
+        ("flip", False, np.bool_, None),
+        ("clip", False, np.bool_, None),
+        ("step", False, np.floating, is_non_negative_value),
+        ("variance", False, np.floating, is_positive_value),
+        ("scale_all_sizes", False, np.bool_, None),
+        ("fixed_ratio", False, np.floating, is_positive_value),
+        ("fixed_size", False, np.floating, is_positive_value),
+        ("density", False, np.floating, is_positive_value),
+        ("min_max_aspect_ratios_order", False, np.bool_, None),
+    ]
+
+    check_valid_attributes("PriorBox", attrs, requirements)
```

Comment:
I think you also need to update init file for opset8
https://github.com/openvinotoolkit/openvino/blob/master/runtime/bindings/python/src/compatibility/ngraph/opset8/__init__.py

Reply:
updated in commit 6f64144